_topic_map.yml

```diff
@@ -1105,6 +1105,8 @@ Topics:
   File: routing-optimization
 - Name: What huge pages do and how they are consumed by apps
   File: what-huge-pages-do-and-how-they-are-consumed-by-apps
+- Name: Using ArgoCD with
```

Comment:
Missing {product-title}?

Reply:
That doesn't work in the topic map. :( I'll fix the typo, though.

Reply:
:) I typed that comment in without realizing which file it was in.